server/server.go

```diff
@@ -102,6 +101,18 @@ func (s *Server) Stop() {
 	manners.Close()
 }
 
+func (s *Server) RefreshMembers() {
+	s.OrgMembers = nil
```

Comment:
If the API fails, it might be fine to continue using the existing list of org members. Are there safety concerns with that approach?

Reply:
Arguable, but I will remove it for now.